export.php

```diff
@@ -70,6 +70,15 @@ function ikonw($zmienna, $encoding)
 
 $now = date('Y-m-d H:i:s');
 
+header('Content-Type: application/xml');
+echo "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\" ?>
+<gkxml version=\"1.0\" date=\"$now\">";
```

Comment:
I dislike abusive escaping, and I suppose [`short-open-tag`](http://php.net/short-open-tag) is ON, so why not:
```php
printf(
  '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<gkxml version="1.0" date="%s">',
  $now
);
```


I'll make you regret to ask for my input™.

Reply:
> "I'll make you regret to ask for my input™"

:angry: